components/Input.js

```diff
@@ -1,11 +1,19 @@
 export default function Input() {
     return `
             <form action="" method="POST" id="bookInput">
-                <input type="text" name="name" placeholder="Title" required>
-                <input type="text" name="author" placeholder="Author" required>
+                <input type="text" name="title" placeholder="Title" required>
+                <input type="text" name="creator" placeholder="Author" required>
                 <input type="text" name="price" placeholder="Price" required>
-                <input type="text" name="pictureUrl" placeholder="Cover image URL" required>
-                <textarea name="title" rows="8" cols="40" name="sellingPoints" placeholder="Blurbs and selling points" reqiured></textarea>
+                <input type="text" name="image" placeholder="Cover image URL" required>
+                <textarea name="sellingPoints" rows="8" cols="40" placeholder="Blurbs and selling points" reqiured></textarea>
```

Comment:
the `selling_points` name should look a little different here (notice the case!)